roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,20 +15,27 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
     docker_image_base = {
         "origin": {
             "repo": "openshift",
             "image": "origin",
+            "has_registry_console": False,
         },
         "openshift-enterprise": {
             "repo": "openshift3",
             "image": "ose",
+            "has_registry_console": True,
         },
     }
 
+    @classmethod
+    def is_active(cls, task_vars):
+        """Skip hosts with unsupported deployment types."""
+        deployment_type = get_var(task_vars, "deployment_type")
```

Comment:
Perhaps here we should be reading `openshift_deployment_type`, though older configurations/inventories will be still defining only `deployment_type`... @sosiouxme, any suggestion?